WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/IoT/WindowsUpdate.cs

```diff
@@ -87,6 +139,12 @@ public class UpdateInstallTimeInfo
             /// </summary>
             [DataMember(Name = "rebootscheduled")]
             public int Rebootscheduled { get; private set; }
+
+            /// <summary>
+            /// Gets the time when a reboot is scheduled. 
+            /// </summary>
+            [DataMember(Name = "rebootscheduledtime")]
+            public int rebootscheduledtime { get; private set; }
```

Comment:
Can this be a TimeSpan or other related object? The user won't be able to tell what units the int is in (ms since epoch?)